runtime/vm/valueTypeHelpers.cpp

```diff
@@ -0,0 +1,35 @@
+#include "j9.h"
+#include "ut_j9vm.h"
```

Comment:
why is this needed?

Reply:
I include `"ut_j9vm.h"` because it seems to allow me using `Assert_VM_notNull`

Reply:
yes thats correct, I missed that